src/sagemaker/deserializers.py

```diff
@@ -154,3 +154,24 @@ def deserialize(self, data, content_type):
             data.close()
 
         raise ValueError("%s cannot read content type %s." % (__class__.__name__, content_type))
+
+
+class JSONDeserializer(BaseDeserializer):
+    """Deserialize data from an inference endpoint into a Python dictionary."""
```

Comment:
nit: I think *technically* it doesn't have to be a Python dictionary, e.g.

```python
>>> json.loads('"foo"')
'foo'
```

Maybe change the docstring to say you're deserializing from JSON instead?